mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -718,6 +719,19 @@ func resourceMongoDBAtlasAdvancedClusterUpdate(ctx context.Context, d *schema.Re
 
 	timeout := d.Timeout(schema.TimeoutUpdate)
 
+	if d.HasChange("advanced_configuration") {
+		ac := d.Get("advanced_configuration")
+		if aclist, ok := ac.([]interface{}); ok && len(aclist) > 0 {
+			advancedConfReq := expandProcessArgs(d, aclist[0].(map[string]interface{}))
+			if !reflect.DeepEqual(advancedConfReq, matlas.ProcessArgs{}) {
```

Comment:
DeelEqual is not normally recommended in Prod (normally only in tests) but I suppose it's ok here